pegasus/sites.v3/code.org/public/prize.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'CS Leaders Prize - $1 Million for U.S. Schools'
+theme: responsive
+social:
+  "og:description": "Tell us how your school will expand computer science, and you could win $10,000 to make it happen!"
+  "og:image": "/images/marketing/hoc2022-cs-leaders-prize-opengraph.png"
+  "og:image:width": 1200
+  "og:image:height": 630
+---
+
+<link href="/css/generated/cs-leaders-prize.css" rel="stylesheet">
+
+# CS Leaders Prize
+
+<header>
+  <div>
+    <h2>Expand computer science at your school, win $10,000</h2>
+    <p>To help more students access computer science, especially students from underserved communities, we're giving away $1 million in awards — that's right, a MILLION dollars. In partnership with DonorsChoose.org, this CS Education Week one middle and high school in every U.S. state and D.C. will be awarded the CS Leaders Prize of $10,000 to help establish computer science education for their students.</p>
+    <a href="#form"><button>Enter to win</button></a>
+  </div>
+  <div>
+    <img src="/images/marketing/hoc2022-cs-leaders-prize-money-bag.png" alt="Icon of a green bag with a purple dollar sign on it surrounded by gold coins">
+  </div>
+</header>
+
+
+## To be eligible a school must commit to:
+
+<ul>
+  <li>Establishing a <strong><em>new</em></strong> <a href="/educate/csp">Computer Science Principles</a>, <a href="/educate/csa">Computer Science A</a>, or <a href="/educate/csd">Computer Science Discoveries</a> course using Code.org.</li>
+  <li>Describe how you plan to continue expanding access to CS in your school and/or district.</li>
+  <li>Provide a short statement of support from district leadership for expanding CS opportunities as a district priority.</li>
```

Comment:
Should we spell out CS? I don't remember when we do and don't spell it out on the website

Reply:
Good call! Eric said: "on any page, first instance should always be computer science, but after that it's okay to shorten to CS" so I'll add "…computer science (CS)" on line 18.